.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Build And Publish Docker Images
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - docker/**
+  release:
+    types: [published]
+  pull_request:
+    paths:
+      - docker/**
+
+jobs:
+  build-publish:
+    name: Build and Push Docker images to Docker Hub
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+       - name: Build all Horovod flavoured PyTorch-Ignite images
```

Comment:
Seems like intend is incorrect

Reply:
Oh, yeah, because of IDE, I'll force push the fix